src/components/dialogs/NewEntryDialog.tsx

```
@@ -0,0 +1,791 @@
+          case NOTION_PROPERTY_TYPES.STATUS:
+            notionProperties[key] = {
+              status: { name: value },
+            };
+            break;
+          case NOTION_PROPERTY_TYPES.DATE:
+            notionProperties[key] = {
+              date: { start: value },
+            };
+            break;
+          case NOTION_PROPERTY_TYPES.MULTI_SELECT:
+            const tags = multiSelectValues[key] || [];
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)